pkg/util/validation/validation.go

```diff
@@ -98,12 +101,13 @@ func IsDNS1123Label(value string) []string {
 		errs = append(errs, MaxLenError(DNS1123LabelMaxLength))
 	}
 	if !dns1123LabelRegexp.MatchString(value) {
-		errs = append(errs, RegexError(dns1123LabelFmt, "my-name", "123-abc"))
+		errs = append(errs, RegexError(dns1123LabelErrMsg, dns1123LabelFmt, "my-name", "123-abc"))
 	}
 	return errs
 }
 
 const dns1123SubdomainFmt string = dns1123LabelFmt + "(\\." + dns1123LabelFmt + ")*"
+const dns1123SubdomainErrorMsg string = "a valid DNS (RFC 1123) subdomain label must consist of lower case alphanumeric character, '-' or '.', and must start/end with an alphanumeric character"
```

Comment:
Pluralize first `characters`: `must consist of lower case alphanumeric characters, ...`